recipes/bareditor/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "bareditor" %}
+{% set version = "0.17" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "6dbf897ff03a75182c23c3367dc110bf676857df9548e4bca6b0b452fdf0f432" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  {{ hash_type }}: {{ hash_value }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - numba
+    - pillow
+    - wxpython
+
+test:
+  imports:
+    - bareditor
+    - bareditor.bareditor
+    - bareditor.imgdata
+
+about:
+  home: https://github.com/Prevalenter/bareditor
+  license: BSD-3-clause
+  license_family: BSD
```

Comment:
You should open an issue upstream that they have an inconstent license in the setup.py

Reply:
The GH issues are disabled for the project, so I went ahead and proposed a fix via a PR: https://github.com/Prevalenter/bareditor/pull/1.